### ISSUE TITLE
Separate font for chorus (or any other environment) 

### ISSUE THREAD

**Issue by xeruf** [ChordPro/chordpro#174]
I have commonly seen a style where the chorus is printed in bold rather than having a sidebar, but I do not see an way to accomplish this with the current options. How about the ability to set fonts for any environment? This would enable many possibilities for custom formatting.

**Comment by sciurius** [comment]
There have been plans for something similiar to this, but I deferred them since it is a bit more than just a font... You would like to control fonts, sizes, colours for lyrics, comments, chords, ... It needs some extra thoughts.
Hint: the forum is a good place to discuss this.

As for your specific question, add this to the config:

````
pdf.chorus.bar.width:0
pdf.chorus.recall.quote:1
pdf.chorus.recall.choruslike:1
````
and try
````
{title: Chorus fonts}}

[A]Hello, World!

{soc}
{textfont Times-Bold}
[E]Dreaming, [A]Dreaming, [B]Just go on
[E]Dreaming, [A]Dreaming, [B]Just go on
{textfont}
{eoc}

[A]Second verse.

{chorus}

[E]The End.
````

**Comment by xeruf** [comment]
Thank you! I totally missed the directions to change fonts & co in the documentation ^^

**Comment by xeruf** [comment]
Regarding the forum: For me it says "[INACTIVE] ChordPro" in the title, so I thought it was not used anymore

**Comment by sciurius** [comment]
Oh? Where does it say INACTIVE? Screenshot?

**Comment by xeruf** [comment]
![image](https://user-images.githubusercontent.com/13354331/133512110-df216a7c-cacf-42bf-a573-e86a75ccabb2.png)
Interface in German, but I guess it conveys the info

**Comment by xeruf** [comment]
Tried the config, but the chorus is still offset to the right even though there is no bar anymore

**Comment by xeruf** [comment]
Oh, I had to also set:
pdf.chorus.indent=0

**Comment by sciurius** [comment]
With what email did you subscribe to the groups.io group?

**Comment by xeruf** [comment]
Oh there is a groups.io group - the issue template links to a google group...

**Comment by sciurius** [comment]
Can you verify? This was changed some time ago.

**Comment by xeruf** [comment]
Yes, I contributed the recent issue template updates but wasn't aware of the group change ;)
